install_template/templates/products/postgis/almalinux-8-or-rocky-linux-8.njk

```diff
@@ -1,2 +1,16 @@
 {% extends "products/postgis/base.njk" %}
 {% set platformBaseTemplate = "almalinux-8-or-rocky-linux-8" %}
+{% block installCommand %}
+
+```shell
+# To install PostGIS 3.2:
+dnf install edb-as<xx>-postgis32
+
+# To install PostGIS 3.1 using EDB Postgres 13-14:
+dnf install edb-as<xx>-postgis3
+
+# To install PostGIS 3.1 using EDB Postgres 11:
```

Comment:
I think this is true for EDB Postgres 11 and 12 both

Reply:
per the repos, 3.1 for epas 12 was not released